### ISSUE TITLE
Idea for additional examples

### ISSUE THREAD

**Issue by gterzian** [automerge/spanreed#1]
1. HTTP servers using an automerge document for peer discovery.
    - Use a bootstrap server where peers would get the initial document
    - Use autosurgeon.

**Comment by eberberich** [comment]
how to use fs_store

**Comment by alexjg** [comment]
You'll need to include the `tokio` feature and then you can use `automerge_repo::tokio::FsStorage::open(<path>)` to create a file system storage.

If you're not using tokio then you'll need to write an implementation of `Storage` which uses your runtime of choice. The `automerge_repo::FsStore` is intended to be wrapped in such an implementation.

**Comment by alexjg** [comment]
Yeah the issue is that it's behind a feature flag, I need to set up some stuff to get docs.rs to generate the docs for it.

**Comment by eberberich** [comment]
Another example that is missing: How to sync a document in a **browser** with a document in a **rust client** (using a central ws server in between. Maybe along with local changes happening on either side (say every few seconds) - and also with getting the current state of the doc in the other repo upon connection established.

We try to build it ourselves - and it's probably 'just there' - but as beginner in Rust AND automerge, it's a bit tough to connect all dots correctly - and we'd appreciate to see such an example

**Comment by eberberich** [comment]
Any hint on such an example?
